### ISSUE TITLE
Remove __MACOSX directory and .DS_Store file

### ISSUE THREAD

**Issue by timmontague** [dhg/Skeleton#178]
```
$ wget -q https://github.com/dhg/Skeleton/releases/download/v2.0/Skeleton.2.0.zip
$ unzip Skeleton.2.0.zip 
Archive:  Skeleton.2.0.zip
   creating: css/
  inflating: css/.DS_Store           
   creating: __MACOSX/
   creating: __MACOSX/css/
  inflating: __MACOSX/css/._.DS_Store  
  inflating: css/normalize.css       
  inflating: css/skeleton.css        
   creating: images/
  inflating: images/favicon.png      
   creating: __MACOSX/images/
  inflating: __MACOSX/images/._favicon.png  
  inflating: index.html              
```

The  `__MACOSX` directory and `.DS_Store` file should probably be removed from the .zip download.

**Comment by dhg** [comment]
Ugh, how did I miss this. Getting on fixing this now.

**Comment by timmontague** [comment]
Thanks! And thank you for writing Skeleton, it's perfect for my needs.

**Comment by dhg** [comment]
For sure! Just released a small point 2.0.1 that _should_ fix this so going to close...but if you get a chance to download (https://github.com/dhg/Skeleton/releases/download/v2.0.1/Skeleton-2.0.1.zip) let me know if that resolved both issues conclusively.

**Comment by timmontague** [comment]
That fixed it. Thanks!

```
$ wget -q https://github.com/dhg/Skeleton/releases/download/v2.0.1/Skeleton-2.0.1.zip
$ unzip Skeleton-2.0.1.zip 
Archive:  Skeleton-2.0.1.zip
   creating: Skeleton 2.0.1/
   creating: Skeleton 2.0.1/css/
  inflating: Skeleton 2.0.1/css/normalize.css  
  inflating: Skeleton 2.0.1/css/skeleton.css  
   creating: Skeleton 2.0.1/images/
 extracting: Skeleton 2.0.1/images/favicon.png  
  inflating: Skeleton 2.0.1/index.html
```

Also thanks for putting it in a directory so it doesn't clobber things when unzipping.
